### ISSUE TITLE
Update colors for accessibility

### ISSUE THREAD

**Issue by codeknitter** [edlio/uniform#137]
# Context

The color palette in uniform was designed before we were focused on accessibility and the color intended for hints doesn't have high enough contrast ratio. Let's fix that and adjust some of the close-by colors as well to keep the colors distinguishable.

# Acceptance Criteria

- [x] change #888 ($grey-5 and $medium-grey) to #767676 - this is the color we should be using for hints
- [x] change #555 ($grey-6) to #444 - this is the color we should be using for body text and is the default font color when no other is provided
- [x] change #333 ($grey-7 and $dark-grey) to #222
- [x] make sure the various elements shown on https://edlio.github.io/uniform/ are calling the variable names ($grey-5, 6, 7) so that they will update automatically with this change.

![revised uniform colors Sept 2019](https://user-images.githubusercontent.com/5893954/65257473-dd23c700-dac6-11e9-8534-785b65f63d98.png)
![uniform- form element colors](https://user-images.githubusercontent.com/5893954/65258514-9e8f0c00-dac8-11e9-9ecb-c04f3fd76343.png)
